src/file.ts

```diff
@@ -72,6 +72,7 @@ export interface GetSignedUrlConfig {
   contentMd5?: string;
   contentType?: string;
   expires: string | number | Date;
+  usableFrom?: string | number | Date;
```

Comment:
Please add a system test to confirm this works with the service.

Reply:
Done - also added conformance test case